src/app/faq-block/faq-block.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+interface Faq {
+  question: string;
+  answer: string;
+}
+
+@Component({
+  selector: 'app-faq-block',
+  templateUrl: './faq-block.component.html',
+  styleUrls: ['./faq-block.component.scss']
+})
+export class FaqBlockComponent implements OnInit {
+
+  @Input() headline: string;
+  @Input() json: string;
+
+  toggle = [];
```

Comment:
If you stick with the current implementation and keep this array, please specify the type of values that are supposed to be stored in the array -> `toggle: boolean[] = []`.

Reply:
It was black magic ... 
I have now created an additional FAQ entry with the "show" attribute